modular_citadel/code/game/gamemodes/gangs/gang_items.dm

```diff
@@ -0,0 +1,410 @@
+/datum/gang_item/weapon/switchblade
+	name = "Switchblade"
+	id = "switchblade"
+	cost = 5
+	item_path = /obj/item/switchblade
+
+/datum/gang_item/weapon/surplus
+	name = "Surplus Rifle"
+	id = "surplus"
+	cost = 6
+	item_path = /obj/item/gun/ballistic/automatic/surplus
+
+/datum/gang_item/weapon/ammo/surplus_ammo
+	name = "Surplus Rifle Ammo"
+	id = "surplus_ammo"
+	cost = 5
+	item_path = /obj/item/ammo_box/magazine/m10mm/rifle
+
+/datum/gang_item/weapon/improvised
+	name = "Sawn-Off Improvised Shotgun"
+	id = "sawn"
+	cost = 5
+	item_path = /obj/item/gun/ballistic/revolver/doublebarrel/improvised/sawn
+
+/datum/gang_item/weapon/ammo/improvised_ammo
+	name = "Box of Buckshot"
+	id = "buckshot"
+	cost = 5
+	item_path = /obj/item/storage/box/lethalshot
+
+/datum/gang_item/weapon/pistol
+	name = "10mm Pistol"
+	id = "pistol"
+	cost = 25
+	item_path = /obj/item/gun/ballistic/automatic/pistol
+
+/datum/gang_item/weapon/ammo/pistol_ammo
+	name = "10mm Ammo"
+	id = "pistol_ammo"
+	cost = 10
+	item_path = /obj/item/ammo_box/magazine/m10mm
+
+/datum/gang_item/weapon/sniper
+	name = "Black Market .50cal Sniper Rifle"
+	id = "sniper"
+	cost = 35
+	item_path = /obj/item/gun/ballistic/automatic/sniper_rifle
+
+/datum/gang_item/weapon/ammo/sniper_ammo
+	name = "Smuggled .50cal Sniper Rounds"
+	id = "sniper_ammo"
+	cost = 15
+	item_path = /obj/item/ammo_box/magazine/sniper_rounds
+
+/datum/gang_item/weapon/ammo/sleeper_ammo
+	name = "Illicit Soporific Cartridges"
+	id = "sniper_ammo"
+	cost = 15
+	item_path = /obj/item/ammo_box/magazine/sniper_rounds/soporific
+
+/datum/gang_item/weapon/machinegun
+	name = "Mounted Machine Gun"
+	id = "MG"
+	cost = 45
+	item_path = /obj/machinery/manned_turret
+	spawn_msg = "<span class='notice'>The mounted machine gun features enhanced responsiveness. Hold down on the trigger while firing to control where you're shooting.</span>"
+
+/datum/gang_item/weapon/machinegun/spawn_item(mob/living/carbon/user, obj/item/device/gangtool/gangtool)
+	new item_path(user.loc)
+	to_chat(user, spawn_msg)
+
+/datum/gang_item/weapon/uzi
+	name = "Uzi SMG"
+	id = "uzi"
+	cost = 50
+	item_path = /obj/item/gun/ballistic/automatic/mini_uzi
+
+/datum/gang_item/weapon/ammo/uzi_ammo
+	name = "Uzi Ammo"
+	id = "uzi_ammo"
+	cost = 20
+	item_path = /obj/item/ammo_box/magazine/uzim9mm
+
+///////////////////
+//EQUIPMENT
+///////////////////
+
+/datum/gang_item/equipment
+	category = "Purchase Equipment:"
+
+
+/datum/gang_item/equipment/spraycan
+	name = "Territory Spraycan"
+	id = "spraycan"
+	cost = 1
+	item_path = /obj/item/toy/crayon/spraycan/gang
+
+/datum/gang_item/equipment/spraycan/spawn_item(mob/living/carbon/user, datum/team/gang/gang, obj/item/device/gangtool/gangtool)
+	var/obj/item/O = new item_path(user.loc, gang)
+	user.put_in_hands(O)
+
+/datum/gang_item/equipment/sharpener
+	name = "Sharpener"
+	id = "whetstone"
+	cost = 3
+	item_path = /obj/item/sharpener
+
+
+/datum/gang_item/equipment/emp
+	name = "EMP Grenade"
+	id = "EMP"
+	cost = 7
```

Comment:
Prob should be lowed to 5

Reply:
EMP grenades are actually really strong, So it stays for the moment, I feel.